README.md

```diff
@@ -2,65 +2,113 @@
 [![dependency status](https://img.shields.io/david/cadre/tcomb-builder.svg?style=flat-square)](https://david-dm.org/cadre/tcomb-builder)
 ![npm downloads](https://img.shields.io/npm/dm/tcomb-builder.svg?style=flat-square)
 
-# Tcomb Form Builder
+# tcomb-builder
+
+_tcomb-builder is an immutable interface for [tcomb-form](https://github.com/gcanti/tcomb-form)_
 
 ## Install
 
 `npm install tcomb-builder tcomb-validation`
 
-## Summary
+## Benefits
+
+1. Immutability
+
+2. Replaces deeply nested JSON options objects in your codebase
+
+3. Yields a standard tcomb type and options JSON object
```

Comment:
you have `JSON options objects` above. and here you have `options JSON object`

I think you refer to it as `option object` in the rest of the doc

Reply:
done